command/server/server.go

```diff
@@ -207,7 +207,13 @@ func setDevFlags(cmd *cobra.Command) {
 }
 
 func runPreRun(cmd *cobra.Command, _ []string) error {
+	// Set the grpc and json ip:port bindings
+	// The config file will have presedence over --flag
+	params.setRawGRPCAddress(helper.GetGRPCAddress(cmd))
+	params.setRawJSONRPCAddress(helper.GetJSONRPCAddress(cmd))
+
 	// Check if the config file has been specified
+	// Config file settings will override --jsonrpc and --gprc flag settings
```

Comment:
You don't need to put the flag names, just for example `JSON-RPC` and `GRPC` address values

Reply:
Fixed dd56594582dbec0c9055c385f1022476d8e34eb9